Adding advice page

Allison-H-H opened the pull request.

twosavoie commented:
Hi @Allison-H-H,

Nice work! 

If you could make one small change in the index.html file. It looks like the formatting has changed and the links are indented just a bit more than they were. If you could reverse the change, keeping the link to your advice file, that would be perfect!

Once you fix that up, you can add, commit, and push and your changes will join your pull request here.
Let us know if you need any help!

xoxo,
Lisa

Allison-H-H commented:
Ok I moved back the indention for the links by a bit.  Not sure what happened as I don't remember changing the indention.  I went ahead and added, committed, and pushed the changes.  Hope it all worked!

redrambles commented:
Hey @Allison-H-H !

Thank you for that change - everything looks good and your code is merged in! 🎉

Cheers,
Ann :)
